add-chain/testdata/superchain/configs/sepolia/expected.yaml

```diff
@@ -0,0 +1,19 @@
+name: awesomechain
+chain_id: 42069
+public_rpc: http://awe.some.rpc
+sequencer_rpc: http://awe.some.seq.rpc
+explorer: https://awesomescan.org
+superchain_level: 2
+batch_inbox_addr: "0xff00000000000000000000000000000000042069"
+genesis:
```

Comment:
I think this file should have some spaces between sections, see the exising config yaml files.

Reply:
Fixed here: https://github.com/ethereum-optimism/superchain-registry/pull/204/commits/7013cd31745c57b2b069cf957329c3d8561445df